k8s/helm-charts/logsearch-helm.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
why not a Makefile?

Reply:
in the end i think we will use that, it is a temporal solution right now. probably using helm from docker would be good idea here as well